### PR TITLE
Add active business days based on creation time

### PR DESCRIPTION
**Pull request by dhaselhan**
* First pass of active days, only considers days from creation and no pauses

### CODE REVIEW

File: alcs-frontend/src/app/shared/card/card.component.scss [hunk 1]
```diff
@@ -18,18 +18,33 @@
...
+  align-items: center;
+}
+
 .assignee {
   border-radius: 50%;
   padding: 6px;
   border: 1px solid #0a5a9c;
 }
 
+.activeDays {
```
Comment (mhuseinov) [inline alcs-frontend/src/app/shared/card/card.component.scss:32]: classes usually use '-' not camelCase  => active-days

File: alcs-api/src/application/application.controller.ts [hunk 2]
```diff
@@ -71,6 +60,11 @@ export class ApplicationController {
       status: app.status.code,
       assigneeUuid: app.assigneeUuid,
       assignee: app.assignee,
+      activeDays: this.businessDayService.calculateDays(
+        new Date(app.auditCreatedBy),
```
[the remark this answers is not in the document]
Comment (dhaselhan) [reply alcs-api/src/application/application.controller.ts:64]: Definitely, next MR will add a new paused / unpaused tracking using a new table

[the remark this answers is not in the document]
Comment (mhuseinov) [reply alcs-api/src/application/application.controller.ts:64]: this is a placeholder for now

File: alcs-frontend/src/app/shared/card/card.component.scss (hunk 1 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/shared/card/card.component.scss]: Good catch! I even used - in the same file >.>
Fixed!

**Review by mhuseinov — APPROVED** [review]
💯

**Pull request merged**